kms/api/CloudKmsTest/CloudKmsTest.cs

```diff
@@ -279,6 +294,124 @@ private bool AssertRoleAndMemberPresent(string response, string role, string mem
             }
             return (foundRole && foundMember);
         }
+
+        private CryptoKeyVersionName BuildAsymmetricKey(KeyRingName keyRingName, String cryptoKey,
+                CryptoKey.Types.CryptoKeyPurpose purpose,
+                CryptoKeyVersion.Types.CryptoKeyVersionAlgorithm algorithm)
+        {
+            KeyManagementServiceClient client = KeyManagementServiceClient.Create();
+            CryptoKey cryptoKeyToCreate = new CryptoKey
+            {
+                Purpose = purpose,
+                VersionTemplate = new CryptoKeyVersionTemplate
+                {
+                    Algorithm = algorithm,
+                    ProtectionLevel = ProtectionLevel.Software,
+                },
+            };
+            CryptoKey key = client.CreateCryptoKey(keyRingName, cryptoKey, cryptoKeyToCreate);
+
+            CryptoKeyVersionName ckvName = CryptoKeyVersionName.Parse(key.Name + "/cryptoKeyVersions/1");

```

Comment:
Instead of doing this, use new CryptoKeyVersionname(key....., 1);

Reply:
Done